bootstrap/httpserver/status.go

```diff
@@ -0,0 +1,110 @@
+package httpserver
+
+import (
+	"encoding/json"
+	"github.com/orbs-network/orbs-network-go/config"
+	"github.com/orbs-network/scribe/log"
+	"net/http"
+	"time"
+)
+
+type Payload struct {
+	Uptime int64
+
+	BlockStorage_BlockHeight   int64
+	StateStorage_BlockHeight   int64
+	BlockStorage_LastCommitted int64
+
+	Gossip_IncomingConnections int64
+	Gossip_OutgoingConnections int64
+
+	Management_LastUpdated  int64
+	Management_Subscription string
+
+	Version config.Version
+}
+
+type StatusResponse struct {
+	Timestamp time.Time
+	Status    string
+	Error     string
+	Payload   Payload
+}
+
+func (s *HttpServer) getStatus(w http.ResponseWriter, r *http.Request) {
+	w.Header().Set("Content-Type", "application/json")
+
+	status := StatusResponse{
+		Timestamp: time.Now(),
+		Status:    s.getStatusWarningMessage(),
+		Payload: Payload{
```

Comment:
this is not what we agreed on ... for now payload will be the current metrics. it will be change in the future